docs/sdk/src/guides/your_first_node.rs

```diff
@@ -1 +1,311 @@
 //! # Your first Node
+//!
+//! In this guide, you will learn how to run a runtime, such as the one created in
+//! [`your_first_runtime`], in a node. Within the context of this guide, we will focus on running
+//! the runtime with an [`omni-node`]. Please first read this page to learn about the OmniNode, and
+//! other options when it comes to running a node.
+//!
+//! [`your_first_runtime`] is a runtime with no consensus related code, and therefore can only be
+//! executed with a node that also expects no consensus ([`sc_consensus_manual_seal`]).
+//! `polkadot-omni-node`'s [`--dev-block-time`] precisely does this.
+//!
+//! > All of the following steps are coded as unit tests of this module. Please see `Source` of the
+//! > page for more information.
+//!
+//! ## Running The Omni Node
+//!
+//! ### Installs
+//!
+//! The `polkadot-omni-node` can either be downloaded from the latest [Release](https://github.com/paritytech/polkadot-sdk/releases/) of `polkadot-sdk`,
+//! or installed using `cargo`:
+//!
+//! ```text
+//! cargo install polkadot-omni-node
+//! ```
+//!
+//! Next, we need to install the [`chain-spec-builder`]. This is the tool that helps us interact
+//! with the genesis related APIs of the runtime, as described in
+//! [`crate::guides::your_first_runtime#genesis-configuration`].
+//!
+//! ```text
+//! cargo install staging-chain-spec-builder
+//! ```
+//!
+//! ### Building Runtime
+//!
+//! Next, we need to build the corresponding runtime that we wish to interact with.
+//!
+//! ```text
+//! cargo build --release -p path-to-runtime
+//! ```
+//! Equivalent code in tests:
+#![doc = docify::embed!("./src/guides/your_first_node.rs", build_runtime)]
+//!
+//! This creates the wasm file under `./target/wbuild/release` directory.
+//!
+//! ### Building Chain Spec
+//!
+//! Next, we can generate the corresponding chain-spec file. For this example, we will use the
+//! `development` (`sp_genesis_config::DEVELOPMENT`) preset.
+//!
+//! Note that we intend to run this chain-spec with `polkadot-omni-node`, which is tailored for
+//! running parachains. This requires the chain-spec to always contain the `--para-id` and a
+//! `--relay-chain` fields, which are provided below.
```

Comment:
```suggestion
//! running parachains. This requires the chain-spec to always contain the `para_id` and a
//! `relay_chain` fields, which are provided below as CLI arguments.
```
Nit: slightly more clear to name them after the chain-spec fields.